gradle/verification-metadata.xml

```diff
@@ -2111,6 +2122,9 @@
          </artifact>
       </component>
       <component group="org.openjfx" name="javafx-base" version="15">
+         <artifact name="javafx-base-15-linux.jar">
```

Comment:
But wouldn't that fail now locally when e.g. building a release binary on macOS?

Reply:
No, this simply adds the expected hash for the linux libs to the list. There is a different hash for `javafx-base-15-mac.jar` / `javafx-base-15-linux.jar` / `javafx-base-15-windows.jar`. Depending on which platform the gradle build is run, a different lib is loaded and a different hash is checked. As long as all 3 are defined in this list, it should be fine.